card_db_src/xx/translation.txt

```diff
@@ -0,0 +1,137 @@
+# Translation Instructions
```

Comment:
I'm surprised to see a .txt version of this too @sumpfork – is your assumption that the Markdown file is not as readable to non-tech people?

Reply:
Not really sure why @wvoigt did that - but yeah, could probably get rid of one?